crates/bevy_ecs/macros/src/component.rs

```diff
@@ -560,6 +568,13 @@ fn parse_component_attr(ast: &DeriveInput) -> Result<Attrs> {
         }
     }
 
+    if attrs.relationship_target.is_some() && attrs.clone_behavior.is_some() {
+        return Err(syn::Error::new(
+                attrs.clone_behavior.span(),
+                format!("A Relationship Target already has it's own clone behavior, please remove `clone_behavior = {}`", attrs.clone_behavior.as_ref().unwrap().to_token_stream()),
```

Comment:
Nit: I don't think you need to provide the token value. Any `clone_behavior` would result in this error. Plus I believe printing the token streams can sometimes look different than the input (at least in the case of a `Custom` behavior).

```suggestion
               "A Relationship Target already has its own clone behavior, please remove `clone_behavior`",
```